File: src/java/5/trunk/src/main/java/com/mycompany/Release3.java
package com.mycompany;

import java.util.List;
import java.util.Random;

import static java.util.Arrays.asList;

/**
 * @author Paul Hammant
 */
public class Release3 extends BranchByAbstractionFactory {
    @Override
    public String getHairColor() {
        List<String> colors = asList("Blonde", "Brown", "Black", "Red");
        return colors.get(new Random().nextInt(colors.size()));
    }
}

File: src/dotnet/3/trunk/Release3.cs
namespace MyCompany;

/// <author>Paul Hammant DevOps, (c) 2018</author>
public class Release3 : IReleaseToggles
{
    public object ChangingHairColor()
    {
        string[] colors = { "Blonde", "Brown", "Black", "Red" };
        return colors[new Random().Next(colors.Length)];
    }
}
